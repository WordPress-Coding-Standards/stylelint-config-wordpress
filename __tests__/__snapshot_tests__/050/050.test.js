'use strict';

const stylelint = require( 'stylelint' ),
	systemTestUtils = require( '../systemTestUtils.js' );

it( '050', () => {
	return stylelint.lint({
		files: [systemTestUtils.caseStylesheetGlob( '050' )],
		configFile: systemTestUtils.caseConfig( '050' ),
	}).then( ({ results }) => {
		expect( systemTestUtils.prepResults( results ) ).toMatchSnapshot();
	});
});
