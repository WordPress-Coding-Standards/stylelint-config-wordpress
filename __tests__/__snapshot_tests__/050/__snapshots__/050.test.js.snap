// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`050 1`] = `
Array [
  Object {
    "deprecations": Array [],
    "errored": true,
    "ignored": undefined,
    "invalidOptionWarnings": Array [],
    "parseErrors": Array [],
    "source": "050/stylesheet.css",
    "warnings": Array [
      Object {
        "column": 1,
        "line": 57,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 85,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 92,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 547,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 840,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 1121,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 1128,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 1251,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 1770,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 117,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 122,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 128,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 133,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 136,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 141,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 156,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 159,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 165,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 191,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 226,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 236,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 240,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 246,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 251,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 255,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 258,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 264,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 270,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 273,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 279,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 284,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 287,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 294,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 298,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 309,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 315,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 392,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 408,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 424,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 447,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 469,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 476,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 481,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 487,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 505,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 524,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 531,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 557,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 586,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 589,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 594,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 653,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 658,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 661,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 665,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 669,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 719,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 752,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 826,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 830,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 853,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 861,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 864,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 896,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 905,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 912,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 916,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 920,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 924,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 931,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 939,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 953,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 959,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 963,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 967,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 970,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 974,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 977,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 980,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1002,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1012,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1035,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1039,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1043,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1050,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1060,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1067,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1075,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1103,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1106,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1109,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1179,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1204,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1223,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1239,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1267,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1271,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1288,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1312,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1328,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1462,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1509,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1535,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1576,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1750,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1873,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1879,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1893,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1898,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1935,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 45,
        "line": 104,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 47,
        "line": 133,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 94,
        "line": 133,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 43,
        "line": 136,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 86,
        "line": 136,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 52,
        "line": 159,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 34,
        "line": 330,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 46,
        "line": 731,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 80,
        "line": 816,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 81,
        "line": 1334,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 88,
        "line": 1413,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 61,
        "line": 1629,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 30,
        "line": 1716,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 2,
        "line": 185,
        "rule": "shorthand-property-no-redundant-values",
        "severity": "error",
        "text": "Unexpected longhand value '5px 0 5px' instead of '5px 0' (shorthand-property-no-redundant-values)",
      },
      Object {
        "column": 3,
        "line": 534,
        "rule": "shorthand-property-no-redundant-values",
        "severity": "error",
        "text": "Unexpected longhand value '0 0 15px 0' instead of '0 0 15px' (shorthand-property-no-redundant-values)",
      },
      Object {
        "column": 2,
        "line": 1231,
        "rule": "shorthand-property-no-redundant-values",
        "severity": "error",
        "text": "Unexpected longhand value '5px 0 0 0' instead of '5px 0 0' (shorthand-property-no-redundant-values)",
      },
      Object {
        "column": 2,
        "line": 1476,
        "rule": "shorthand-property-no-redundant-values",
        "severity": "error",
        "text": "Unexpected longhand value '15px 0 0 0' instead of '15px 0 0' (shorthand-property-no-redundant-values)",
      },
    ],
  },
]
`;
